Tests/Functional/Drivers/Queue/AbstractQueueDriverTest.php

```diff
@@ -54,147 +50,18 @@ protected function tearDown()
     public function testSendShouldNotChangeMessage(MessageInterface $msg)
     {
         $clone = clone $msg;
-
+        if (!$this->driver->isConnected()) {
+            $this->driver->connect();
+        }
```

Comment:
Is this piece of code needed if in the setup we are already executing
```
$this->driver->connect();
```

Reply:
Yes, this class already existed when I implemented the tests to PhpAmqpLib. However, I think it's good to start the tests connected.

Reply:
The code will never go inside the if statement because the driver connection is setup in the **setUp** method of the test. For me this doesn't give extra value and can be deleted

Reply:
Fixed.

Reply:
:-1: Forgot to push @brunohsouza ? I still see this if in the latest version

Reply:
In my changes I have deleted this condition

Reply:
It's the commit where I removed this condition in 04/03/2020: 8279a347c6e492cdb37ad9a734232a9564a49890. Maybe you forgot to pull.

Reply:
In feature/exception-handler I have deleted this, so I resolve the conversation